src/components/Navbar/Navbar.tsx

```diff
@@ -4,9 +4,13 @@ import { Link, NavLink, useNavigate } from "react-router-dom";
 import Logo from "../../assets/logo.svg";
 import loggedProfile from "../../assets/man.png";
 import cartImage from "../../assets/cartImage.svg";
+import WishListIcon from "../../assets/WishlistIcon.png";
 import UnknownProfile from "../../assets/profile.svg";
 import "./Navbar.scss";
 const Navbar = () => {
+  const WishCars = useSelector((state: any) => state.wishListReducer.wishCars);
```

Comment:
Only React components have CamelCaseing starting with upper case, rest all should have camelCase starting with lower case